transmart-core-api/src/main/groovy/org/transmartproject/core/audit/AuditLogger.groovy

```diff
@@ -0,0 +1,10 @@
+package org.transmartproject.core.audit
+
+import javax.servlet.http.HttpServletRequest
+
+interface AuditLogger {
+
+    boolean getEnabled()
+
+    def report(Map<String, Object> params, String event, HttpServletRequest request)
+}
```

Comment:
Small stylistic feedback: turn on automatic empty line adding in IntelliJ https://www.jetbrains.com/help/idea/code-style-java.html
Why it is important: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#729795